script/install.sh

```diff
@@ -421,6 +421,16 @@ resolve_latest() {
             logV "Determining latest ldc-beta version ($url)."
             COMPILER="ldc-$(fetch $url)"
             ;;
+        ldc-nightly)
+            local CIurl=https://github.com/ldc-developers/ldc/releases/tag/CI
+            logV "Finding latest ldc-nightly binary package (at $CIurl)."
+            local package="$(curl -s $CIurl | grep -o "ldc2-[0-9a-f]*-$OS-$ARCH-[0-9]*.tar.xz" | head -n 1)"
```

Comment:
Shellcheck error: 

```
In script/install.sh line 427:
            local package="$(curl -s $CIurl | grep -o "ldc2-[0-9a-f]*-$OS-$ARCH-[0-9]*.tar.xz" | head -n 1)"
                  ^-- SC2155: Declare and assign separately to avoid masking return values.
```

Reply:
See: https://github.com/koalaman/shellcheck/wiki/SC2155

Reply:
How do you want me to fix this?
`local -r` silences the warning; i don't particularly care about error codes, because we handle the error elsewhere. Still, it might be nice for the user to see the raw error.. ?

Reply:
I think both `local -r` and using a separate line for the definition work though the latter seems to be the more prevalent style in this script, so I would opt for that one.

Reply:
Done